lib/ios/RNNBackButtonOptions.m

```diff
@@ -11,11 +11,36 @@ - (void)applyOn:(UIViewController *)viewController {
 		
 		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithImage:image style:UIBarButtonItemStylePlain target:nil action:nil];
 		[self setBackItem:backItem onViewController:viewController];
-	} else if (self.title) {
-		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithTitle:self.title
+	} else {
+		NSString *title;
+		
+		title = self.title ? self.title : viewController.navigationItem.title;
```

Comment:
Great! You can also combine this to one line `NSString *title = self.title ? self.title : viewController.navigationItem.title;`

Reply:
I think this is wrong - it sets the back button title to the value of the current screen's title.

Reply:
@chrise86  So basically solution is to always use `self.title`?

Reply:
@henrikra kinda... what that does mean though, is that you will always have to pass a title to the back button in order. If you don't, then you will just get the back arrow alone.

Reply:
Please can you make PR out of this because this one died I guess :D